### PR TITLE
Added Input list modal rating components

### PR DESCRIPTION
**Pull request by Mohitk391**

### CODE REVIEW

File: components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.modal-box {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.modal-content {
+    background-color: #fefefe;
```
Comment (JuvitaSaini09) [inline components/modal/modal.css]: u can use variables

File: components/modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    margin: auto;
+    padding: 20px;
```
Comment (JuvitaSaini09) [inline components/modal/modal.css]: use rem

File: components/rating/rating.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    color: var(--bg-secondary);
+}
+
+.rating :not(:checked) > label {
+    float:right;
+    width:1em;
+    overflow:hidden;
+    white-space:nowrap;
+    cursor:pointer;
+    font-size:30px;
```
Comment (JuvitaSaini09) [inline components/rating/rating.css]: use rem

**Comment by JuvitaSaini09** [comment]
your code looks good to merge , I have recommended  small changes.Also,The  the image of arrow button in  Outlined Buttons was not loading .

**Review by JuvitaSaini09 — APPROVED** [review]
I am approving,avoid pixel rest code looks good to me

File: components/list/list.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+.list-circle{
+    list-style-type: circle;
+}
+
+.list-none{
+    list-style: none;
+}
+.list-stack{
+    list-style: none;
+
```
Comment (nikhiljugale007) [inline components/list/list.css:12]: you are repeating same property, u should use same class.

**Review by nikhiljugale007 — COMMENTED** [review]
In input component error input turns off as we start typing.
Modal should appear at center of screen or try to give some padding top, it is currently stuck at top.

suggestion: Try to format your code before pushing.

File: components/modal/modal.css [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    display: none; 
+    position: fixed; 
+    z-index: 1; 
+    left: 0;
+    top: 0;
+    width: 100%; 
+    height: 100%; 
+    overflow: auto; 
+    background-color: rgb(0,0,0); 
+    background-color: rgba(0,0,0,0.4);
```
Comment (anasm11) [inline components/modal/modal.css]: better to use variables for reusability.

**Review by anasm11 — COMMENTED** [review]
checkbox & radio lists have very light color for text.
Rest looks good.

**Review by anasm11 — APPROVED** [review]

**Pull request merged**